app/org/sagebionetworks/bridge/BridgeUtils.java

```diff
@@ -242,5 +242,25 @@ public static int getIntOrDefault(String value, int defaultValue) {
             throw new BadRequestException(value + " is not an integer");
         }
     }
-    
+
+    /**
+     * Creates a new copy of the map, removing any entries that have a null value (particularly easy to do this in
+     * JSON).
+     */
+    public static <K,V> Map<K,V> withoutNullEntries(Map<K,V> map) {
```

Comment:
Would it be desirable to use this setting across the board?

https://fasterxml.github.io/jackson-databind/javadoc/2.6/com/fasterxml/jackson/databind/SerializationFeature.html#WRITE_NULL_MAP_VALUES